.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 4
```

Comment:
question: Looks like these will be applied to all file types. Is that what we want to do, or do we want to apply these two settings to python files only?

Reply:
good call! i'll see how to do that

Reply:
https://github.com/NLeSC/python-template/blob/b60acef6f8bc10b2d0aac77a82a09969e5ca70b0/.editorconfig#L14

Reply:
thanks!